source/includes/language-compatibility-table-pymongo.rst

```diff
@@ -0,0 +1,517 @@
+
+   * - 4.2
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+
+   * - 4.1 [#three-six-compat]_
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+     -
+     -
+     -
+     -
+     -
+
+   * - 4.0
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+     -
+     -
+     -
+     -
+     -
+
+   * - 3.13
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+     - ✓
+     -
+     -
+
+   * - 3.12
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+     - ✓
+     -
+     -
+
+   * - 3.11
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+     - ✓
+     -
+     -
+
+   * - 3.10
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+     - ✓
+     -
+     -
+
+   * - 3.9
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+     - ✓
+     -
+     -
+
+   * - 3.8
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+     - ✓
+     -
+     -
+
+   * - 3.7
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+     - ✓
+     -
+     -
+
+   * - 3.6
+     -
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     -
+     - ✓
+     -
+     -
+
+   * - 3.5
+     -
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+     -
+
+   * - 3.4
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+     -
+
+   * - 3.3
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+     -
+
+   * - 3.2
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+
+   * - 3.1
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+
+   * - 3.0
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+
+   * - 2.9
+     -
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - 2.8
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - 2.7
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+.. [#ssl-4.0-issue] Versions of Python 3.10 and later are not compatible with
+   TLS/SSL for versions of MongoDB 4.0 and earlier. For more information, see the
+   :ref:`TLS guide <pymongo-python-3.10-tls>`.
+.. [#three-six-compat] Pymongo 4.1 requires Python 3.6.2 or later.
+
+.. note::
+
+   PyPy3 is a Python 3.2-compatible alternative interpreter.
+
+Python 2
+~~~~~~~~
+
+.. list-table::
```

Comment:
If you have any ideas on why this table's columns look weird, please let me know

Reply:
If referring to the column spacing, you can provide hints on the spacing by  using the  `widths` attribute ([Example](https://github.com/mongodb/docs-kafka-connector/blob/master/source/source-connector/configuration-properties/change-stream.txt#L35)).
Add the percentage for each column, e.g.

```
   :widths: 10 30 30 30
```

Reply:
Thanks. That didn't fix it in the way I was hoping; I think the weirdness is a product of the table automatically placing the checkmark inline with the start of the column header. The only fix I can think of is manually putting the header text on two lines (`Python 2.5,\nJython 2.5`), but I couldn't make that work. Not a big deal.

Reply:
Oh I see. If the line break didn't render, I think you can use the "|" block operator, e.g.:

```
- | Python 2.5,
  |
  | Jython 2.5
```